openedx_webhooks/tasks/github.py

```diff
@@ -1,30 +1,34 @@
 # coding=utf-8
 from __future__ import unicode_literals, print_function
 
+from datetime import date
 import json
 import re
-from datetime import date
 
-from iso8601 import parse_date
 from flask import render_template, render_template_string
+from iso8601 import parse_date
 from urlobject import URLObject
+import requests
 
 from openedx_webhooks import sentry, celery
-from openedx_webhooks.tasks import logger
-from openedx_webhooks.oauth import github_bp, jira_bp
 from openedx_webhooks.info import (
-    get_people_file, is_internal_pull_request, is_contractor_pull_request,
+    get_people_file,
     is_beta_tester_pull_request,
+    is_contractor_pull_request,
+    is_internal_pull_request,
 )
-from openedx_webhooks.utils import memoize, paginated_get
 from openedx_webhooks.jira_views import get_jira_custom_fields
+from openedx_webhooks.oauth import github_bp, jira_bp
+from openedx_webhooks.tasks import logger
+from openedx_webhooks.tasks.utils import log_error, log_info, log_request_response
```

Comment:
I'd have though the tasks package was just for tasks, and that utils for tasks could go into the existing utils module, but I guess people differ on that.